### ISSUE TITLE
FileReader.readAsBinaryString is not being reported as unsupported in IE

### ISSUE THREAD

**Issue by falsyvalues** [amilajack/eslint-plugin-compat#381]
Hello 👋 ,

`FileReader.readAsBinaryString` method is not supported under Internet Explorer and it should be reported.

**Plugin version**: eslint-plugin-compat@3.8.0
**Settings**:
```json
{
  "settings": {
    "polyfills": [
      "Array.from",
      "fetch",
      "Object.assign",
      "Object.entries",
      "Promise",
      "Symbol",
      "URL"
    ]
  }
}

```
**Reference**:
- https://caniuse.com/#feat=filereader
- https://developer.mozilla.org/en-US/docs/Web/API/FileReader#Browser_compatibility
